src/workloads/query/TimeseriesTSBSOptimizations.yml

```diff
@@ -0,0 +1,166 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query-integration"
+Description: |
+  This workload runs different time-series optimizations with the tsbs dataset. During development, we targeted optimizations
+  that are most relevant to pushdown time-series queries to SBE. Further optimizations should be added to this workload.
+  The data is preloaded by the dsi configuration. See 'configurations/test_control/test_control.tsbs_query.yml' for all 
+  the details. There are 20736000 documents in the collection.
```

Comment:
I'd collect and add here a few more statistics about the dataset: 
- the number of buckets, 
- the p50 and p80 of the number of documents per bucket 
- how many buckets are v1 vs v2, 
- what % of buckets a time window of 1 hour is expected to hit, 
- what % of buckets a single value of a measurement is expected to hit.

I'd also mention which field is meta and which field is time, the bounds on the time, and which indexes exist. Basically, do not expect the future maintainers to already know the particulars of the dataset (or know where to find this information). Collecting and outlining statistics like this also helps you, the author of the workload, to validate your assumptions about the perf characteristics of the queries.